content/pages/sprint/2024/08-coscup.rst

```diff
@@ -0,0 +1,229 @@
+===========================================
+Scisprint 2024 August in COSCUP
+===========================================
+
+:date: 2024-07-24 20:00
+:url: sprint/2024/08-coscup
+:save_as: sprint/2024/08-coscup.html
+
+Date
+-----
+
+* Date: 3rd August, Saturday, 2024
```

Comment:
Most of our main events are on 8/3, but COSCUP is a two-day event. Do we need to change the event time to 8/3~8/4?

Reply:
I think we do not need to change here, the track will only happen on 8/3  so this is correct.